tests/AbstractDoctrineAnnotationFixerTestCase.php

```diff
@@ -73,12 +73,12 @@ protected static function createTestCases(array $commentCases): array
                 isset($commentCase[1]) ? self::withMethodDocBlock($commentCase[1]) : null,
             ];
 
-            $cases[] = [
+            $cases[$commentCase[0]] = [
                 self::withWrongElementDocBlock($commentCase[0]),
             ];
         }
 
-        return $cases;
```

Comment:
Maybe change all `return self::createTestCases([` into `yield from self::createTestCases([` and than `yield` here directly, saving the array creating and array copy.